WordPress/src/main/java/org/wordpress/android/ui/accounts/AbstractFragment.java

```diff
@@ -63,12 +63,12 @@ protected boolean onDoneEvent(int actionId, KeyEvent event) {
         return false;
     }
 
-    protected boolean didPressNextKey(int actionId, KeyEvent event) {
+    public static boolean didPressNextKey(int actionId, KeyEvent event) {
```

Comment:
Oops, sorry, the `AbstractFragment` is not needed anymore in the context of this PR so, I'll revert its changes. In effect, I don't want this PR to be about AbstractFragment at all. Done with d3df0b0.

Sorry for the noise here :(